app/src/main/res/layout/more_movies_item.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="wrap_content"
+    android:layout_height="wrap_content"
+    android:layout_marginTop="@dimen/dp_10"
+    android:padding="@dimen/dp_5"
+    >
+    <ImageView
+        android:id="@+id/image_more_poster_item"
+        android:layout_width="@dimen/dp_120"
+        android:layout_height="@dimen/dp_178"
+        android:contentDescription="@string/poster_item_description"
+        android:background="@color/colorBackground"
+        />
+    <TextView
+        android:id="@+id/text_more_name_item"
+        android:layout_width="@dimen/dp_120"
+        android:layout_height="@dimen/dp_33"
+        android:layout_below="@+id/image_poster_item"
```

Comment:
WARNING: [UnknownIdInLayout] The id "`image_poster_item`" is not referring to any views in this layout
 The `@+id/` syntax refers to an existing id, or creates a new one if it has not already been defined elsewhere. However, this means that if you have a typo in your reference, or if the referred view no longer exists, you do not get a warning since the id will be created on demand.

This is sometimes intentional, for example where you are referring to a view which is provided in a different layout via an include. However, it is usually an accident where you have a typo or you have renamed a view without updating all the references to it.